projects/unit/configs/all/shared_dec.yaml

```diff
@@ -0,0 +1,196 @@
+      dilation: true
+      use_task_embedding_in_encoder: true
+      use_task_embedding_in_lang_encoder: true
+    bert_config:
+      bert_model_name: bert-base-uncased
+    losses:
+    - logit_bce
+    # initialize the ResNet convnet backbone from DETR
+    detr_ckpt_path: https://dl.fbaipublicfiles.com/detr/detr-r50-dc5-f0fb7ef5.pth
+    detr_ckpt_load_backbone_only: true
+
+dataset_config:
+  vqa2:
+    return_features_info: true
+    use_features: false
+    use_images: true
+    processors:
+      text_processor:
+        type: bert_tokenizer
+        params:
+          tokenizer_config:
+            type: bert-base-uncased
+            params:
+              do_lower_case: true
+          mask_probability: 0
+          max_seq_length: 25
+      detr_image_processor:
+        type: detr_image
+        params:
+          image_size: 800
+          max_size: 1333
+    images:
+      train:
+      - coco/defaults/images/trainval2014
+      - coco/defaults/images/trainval2014
+      - coco/defaults/images/trainval2014
+      val:
+      - coco/defaults/images/trainval2014
+      test:
+      - coco/defaults/images/test2015
+    annotations:
+      train:
+      - vqa2/defaults/annotations/imdb_train2017.npy
+      - vqa2/defaults/annotations/imdb_vg_karpathytrain_not_in_val2017.npy
+      - vqa2/defaults/annotations/imdb_vg_karpathytest_not_in_val2017.npy
+      val:
+      - vqa2/defaults/annotations/imdb_val2017.npy
+      test:
+      - vqa2/defaults/annotations/imdb_test2015.npy
+  # hateful_memes:
+  #   use_features: false
+  #   use_images: true
+  #   zoo_requirements: []
+  #   annotations:
+  #     train:
+  #     - /private/home/dkiela/data/datasets/hm-final-unseen/train.jsonl
+  #     val:
+  #     - /private/home/dkiela/data/datasets/hm-final-unseen/dev.jsonl
+  #     test:
+  #     - /private/home/dkiela/data/datasets/hm-final-unseen/test.jsonl
+  #   images:
+  #     train:
+  #     - /private/home/dkiela/data/datasets/hm-final-unseen
+  #     val:
+  #     - /private/home/dkiela/data/datasets/hm-final-unseen
+  #     test:
+  #     - /private/home/dkiela/data/datasets/hm-final-unseen
+  #   processors:
+  #     detr_image_processor:
+  #       type: detr_image
+  #       params:
+  #         image_size: 224
+  #         max_size: 224
+  visual_entailment:
+    use_features: False
+    zoo_requirements: []
+    annotations:
+      train:
+      - /private/home/asg/data/visual_entailment/defaults/annotations/snli_ve_train.jsonl
+      val:
+      - /private/home/asg/data/visual_entailment/defaults/annotations/snli_ve_dev.jsonl
+      test:
+      - /private/home/asg/data/visual_entailment/defaults/annotations/snli_ve_test.jsonl
+    images:
+      train:
+      - /private/home/asg/data/visual_entailment/defaults/images/flickr30k_images
+      val:
+      - /private/home/asg/data/visual_entailment/defaults/images/flickr30k_images
+      test:
+      - /private/home/asg/data/visual_entailment/defaults/images/flickr30k_images
```

Comment:
These need to change to relative paths for cache

Reply:
(changed to relative paths)